### PR TITLE
dotlocalctl 0.0.2 (new formula)

### PR DESCRIPTION
**Pull request by blackmann**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/d/dotlocalctl.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Dotlocalctl < Formula
+  desc "Backend CLI for DotLocal"
+  homepage "https://degreat.co.uk/dotlocal/cli"
+  url "https://degreat.co.uk/packages/dotlocalctl-0.0.2.tar.gz"
```
Comment (chenrui333) [inline Formula/d/dotlocalctl.rb:4]: is this open source project? if so, where is the code?

Comment (blackmann) [reply Formula/d/dotlocalctl.rb:4]: No, it's not open source. Is it required to be open source?

Comment (SMillerDev) [reply Formula/d/dotlocalctl.rb:4]: Yes, https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Comment (alebcay) [reply Formula/d/dotlocalctl.rb:4]: To be clear, you can still make your software available through Homebrew regardless of its license by hosting the formula in your own [tap](https://docs.brew.sh/Taps), basically a third-party repository that the `brew` tool can install software from.

But to consider software for inclusion in _this_ repo (homebrew-core), the software must be open-source, per the link that Sean shared.

**Pull request closed without merging**